Makefile

```
@@ -159,8 +159,8 @@ release/publish: install/goreleaser
 
 .PHONY: update-gql-schema
 update-gql-schema:
-	@go run ./cmd/gqltool/main.go > ./client/graphql/schema/introspection_query_result.json
-	@cd ./client/graphql/schema && npm run convert
+	@go run ./cmd/gqltool/main.go > ./internal/client/graphql/schema/introspection_query_result.json
```

I think it would be fine to include also the `npm install`, it failed to me on the first try. I had to figure out.

```suggestion
  @npm install --prefix internal/client/graphql/schema
  @go run ./cmd/gqltool/main.go > ./internal/client/graphql/schema/introspection_query_result.json
  @cd ./internal/client/graphql/schema && npm run convert
```

---

Done